parse/GreaterThan.ts

```diff
@@ -1,7 +1,8 @@
 import { GreaterThan } from "../GreaterThan"
-import { add } from "./parse"
+import { add, parseExpression } from "./parse"
 
 add(source => {
-	const fetched = source.fetchIf(">", "any")
-	return fetched && new GreaterThan(isNaN(+fetched[1].value) ? fetched[1].value : +fetched[1].value)
+	const fetched = source.fetchIf(">")
+
+	return fetched && new GreaterThan(parseExpression(source))
```

Comment:
Can be written in one line:
return source.fetchIf(">") && new GreaterThan(parseExpression(source))